lib/parsing/src/query_string.rs

```diff
@@ -0,0 +1,111 @@
+use crate::value::Value;
+use bytes::Bytes;
+use std::collections::BTreeMap;
+
+pub fn parse(bytes: &Bytes) -> Result<Value, String> {
```

Comment:
Just a general curiosity, but I'm wondering if there was specific motivation for a home-brew version of query string parsing vs. reaching for something off-the-shelf, like [serde_urlencoded](https://docs.rs/crate/serde_urlencoded/0.7.0)?

Reply:
Ah, good point, I believe something off-the-shelf could also work since DD support pretty standard behaviour. Will double-check and replace with the library one then.